src/vg.cpp

```diff
@@ -2162,6 +2181,124 @@ void VG::from_gfa(istream& in, bool showp) {
 		}
 
 }
+    static
+    void
+    triple_to_vg(void* user_data, raptor_statement* triple)
+    {
+        VG* vg = ((std::pair<VG*, Paths*>*) user_data)->first;
+        Paths* paths = ((std::pair<VG*, Paths*>*) user_data)->second;
+        const string vg_ns ="<http://example.org/vg/";
```

Comment:
No magic at all. But best to pick uri to be decided in pull vg#217

In src/vg.cpp <https://github.com/vgteam/vg/pull/233#discussion_r54221005>:

> @@ -2162,6 +2181,124 @@ void VG::from_gfa(istream& in, bool showp) {
>  		}
>
>  }
> +    static
> +    void
> +    triple_to_vg(void* user_data, raptor_statement* triple)
> +    {
> +        VG* vg = ((std::pair<VG*, Paths*>*) user_data)->first;
> +        Paths* paths = ((std::pair<VG*, Paths*>*) user_data)->second;
> +        const string vg_ns ="<http://example.org/vg/";

Can we change this now to http://github.com/vgteam/vg, or is there more
magic involved in preparing such a URI prefix before we can set it thusly?

—
Reply to this email directly or view it on GitHub
<https://github.com/vgteam/vg/pull/233/files#r54221005>.